### PR TITLE
docs(breakpoints): create docs for breakpoints implementation

### PR DESCRIPTION
**Pull request by pedrohmorais**
## Description
https://jirasoftware.catho.com.br/browse/QTM-90

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### Browsers review
- [ ] Layout review
  - [ ] Edge
  - [ ] IE 11
  - [ ] Firefox
  - [ ] Chrome
  - [ ] Safari
  - [ ] Mobile
- [ ] Ux review validation

### CODE REVIEW

File: stories/Breakpoints/Breakpoints.story.jsx [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+      styled components.
+      <br />
+      You can create different style rules following the Quantum breakpoints.
+    </Heading>
+
+    <StoryContainer>
+      <Title as="h2">How to use</Title>
+
+      <p>
+        First import `BREAKPOINTS` from `shared`. Second import `query` function
```
Comment (rapahaeru) [inline stories/Breakpoints/Breakpoints.story.jsx]: Where there are the crase, how about using a `<code>` tag?
Only to keep the pattern.

ex:
now:
 ```First import `BREAKPOINTS` from `shared`.```
changed:
```First import <code>BREAKPOINTS</code> from <code>shared</code>.```


Resut

Before:
Captura de tela de 2019-11-27 11-12-37

after:
Captura de tela de 2019-11-27 11-12-14

File: stories/Breakpoints/Breakpoints.story.jsx [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+        from `Grid` shared.
+      </p>
+      <CodeExample
+        code={`import { BREAKPOINTS } from '@catho/quantum/shared';
+import { query } from '@catho/quantum/Grid/sub-components/shared';
+`}
+      />
+      <p>
+        Create a constant variable calling `query` with `BREAKPOINTS` as
+        parameter. After, create a styled component and introce the variable. In
```
Comment (rapahaeru) [inline stories/Breakpoints/Breakpoints.story.jsx]: Did you mean introduce?

File: stories/Breakpoints/Breakpoints.story.jsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+  \`}
+
+  \${mediaQueries.xlarge\`
+    background-color: orange;
+  \`}
+\`;
+      `}
+      />
+
+      <p>Test the div in differents sizes:</p>
```
Comment (rapahaeru) [inline stories/Breakpoints/Breakpoints.story.jsx]: wow, it was show of ball!

**Comment by rapahaeru** [comment]
Its new form me, but:
Captura de tela de 2019-11-27 11-23-57

His claiming for do not use a UL inside a P, i suggest this:

Captura de tela de 2019-11-27 11-26-42

File: stories/Breakpoints/Breakpoints.story.jsx (hunk 2 above)
Comment (pedrohmorais) [reply stories/Breakpoints/Breakpoints.story.jsx]: Yep

File: stories/Breakpoints/Breakpoints.story.jsx (hunk 1 above)
Comment (pedrohmorais) [reply stories/Breakpoints/Breakpoints.story.jsx]: Done

**Pull request merged**